bootstrap/aws/util.sh

```diff
@@ -75,7 +75,7 @@ ansible_playbook_run() {
     ANSIBLE_SSH_ARGS="-o UserKnownHostsFile=/dev/null -o ControlMaster=auto -o \
     ControlPersist=60s -F $APOLLO_ROOT/terraform/aws/ssh.config -q" \
     ansible-playbook --user=ubuntu --inventory-file=$APOLLO_ROOT/inventory/aws \
-    --extra-vars "consul_atlas_infrastructure=${ATLAS_INFRASTRUCTURE} \
+    ${ANSIBLE_LOG} --extra-vars "consul_atlas_infrastructure=${ATLAS_INFRASTRUCTURE} \
```

Comment:
if we fall into the ```error)``` case above and nothing is set for ```ANSIBLE_LOG``` will we get a warning here because the variable is undefined?